setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+with open(path.join(here, 'README.md')) as f:
+    description = f.read()
+
+setup(
+    name='cinch',
+    version='0.1.6',
```

Comment:
Does this follow [semantic versioning](http://semver.org/)?  It looks like it, but I just want to be sure.